talpid-core/src/firewall/linux.rs

```diff
@@ -65,11 +65,10 @@ lazy_static! {
     static ref FORWARD_CHAIN_NAME: CString = CString::new("forward").unwrap();
     static ref PREROUTING_CHAIN_NAME: CString = CString::new("prerouting").unwrap();
 
-    /// We need two separate tables for compatibility with older kernels (holds true for kernel
-    /// version 4.19 but not 5.6), where the base filter type may not be `nftnl::ChainType::Route`
-    /// or `nftnl::ChainType::Nat` for inet tables.
-    static ref MANGLE_TABLE_NAME_V4: CString = CString::new("mullvadmangle4").unwrap();
-    static ref MANGLE_TABLE_NAME_V6: CString = CString::new("mullvadmangle6").unwrap();
+    /// NOTE: The base filter type may not be `nftnl::ChainType::Route` or `nftnl::ChainType::Nat`
+    /// for inet tables when running older kernels. It's been confirmed to work on 5.6, and not work
+    /// on 4.19.
+    static ref MANGLE_TABLE_NAME: CString = CString::new("mullvadmangle").unwrap();
```

Comment:
We should remove the table completely and just use `mullvad` here.

Reply:
Yeah, if we can get away with fewer tables, we should do that!